sql/src/main/java/io/crate/metadata/Functions.java

```diff
@@ -85,7 +84,7 @@ public FunctionImplementation get(FunctionIdent ident) throws IllegalArgumentExc
             return implementation;
         }
 
-        DynamicFunctionResolver dynamicResolver = functionResolvers.get(ident.name());
+        DynamicFunctionResolver dynamicResolver = functionResolvers.get(ident.name().toLowerCase(Locale.ENGLISH));
```

Comment:
As discussed, please do this change in a separate commit / PR. And I'm not sure if this is even the right place to do the lowercase.